b2sdk/sync/folder.py

```diff
@@ -294,7 +301,7 @@ def all_files(self, reporter, policies_manager=DEFAULT_SCAN_MANAGER):
             if policies_manager.should_exclude_file(file_name):
                 continue
 
-            if current_name != file_name and current_name is not None:
+            if current_name != file_name and current_name is not None and current_versions:
```

Comment:
that's worth unit testing

Reply:
test_exclude_modified_multiple_versions

Reply:
I added one more to better cover if from line 325